style.css

```diff
@@ -33,101 +52,175 @@ a:active {
   color:#4d4d4d;
 }
 
-body {
-  margin: 0;
-  font-family: "Montserrat",Verdana,Helvetica,sans-serif;
-  color: black;
-  font-size: 11px;
-}
-
-header {
-  overflow: hidden;
-  padding: 2px 150px 0px 150px;
-  background-color: aliceblue;
+.user-service {
+  display: flex;
+  justify-content: space-between;
+  padding: 9px 140px 0px 140px;
 
 }
 
-#top-left.a {
-  float: left;
+.catalog-store {
+  display: flex;
+  justify-content: flex-start;
 }
 
-#top-right {
-  float: right;
+#user-profile {
+  display: flex;
+  justify-content:flex-end;
   color: #4d4d4d;
 }
 
-#middle {
+.website-logo {
   display:flex;
   justify-content: center;
 }
 
 .logo {
   background: no-repeat;
-  display: block;
+  display: flex;
   width: 376px;
   height: 41px;
   margin: 0;
   padding: 0 0 0 18px;
   background-position: center center !important;
 }
 
-#bottom-left > a {
+.customer-service{
+  display: flex;
+  justify-content: space-between;
+  padding: 2px 140px 22px 140px;
+}
+
+.registry-service{
+  display: flex;
+  justify-content: flex-start;
+}
+
+.registry-service a {
   font-size: 14px;
-  padding-right: 13px;
-  color: black;
   font-weight: 500;
+  color: black;
+  padding-right: 13px;
+
 }
 
-#bottom-left > a:hover {
+.registry-service > a:hover {
   text-decoration: none;
 
 }
 
-#search-box {
-  bottom: 9px;
-  position: relative;
-  margin-top: -16px;
-  width: 240px;
+.search-box {
+  display: flex;
+  justify-content: flex-end;
+  bottom: 10px;
+  position: sticky;
+  width: 300px;
+  height: 40px;
 }
 
 #header-search-box {
   width: 100%;
-  height: 40px;
-  padding-right: 28px;
+  padding-right: 25px;
   border-radius: 6px;
-  background-size: 18px;
 }
 
+
+/* nav bar */
+
+nav {
+  width: 100%;
+  display: flex; /* Use flexbox to layout the items horizontally */
+  justify-content: center; /* Add spacing between each item */
+  background-color: white; /* Set a background color */
+  color: black; /* Set the font color */
+  font-size: 15px;
+  text-align: center;
+  padding: 0;
+  list-style-type: none;
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+  margin: 0;
+
+}
+
+ul {
```

Comment:
it's  general, 
 use id or class as a wrapper  for all this will destroy the project